setup.py

```diff
@@ -33,6 +33,7 @@
         "enum34; python_version<'3.4'",
         # Install marshmallow with 'reco' (recommended) extras to ensure a
         # compatible version of python-dateutil is available
+        "attr",
```

Comment:
`attr` is, confusingly, another package:
```suggestion
        "attrs",
```

Reply:
Well spotted - this only worked in tests as `pytest` depends on `attrs`.